server/plugin/command.go

```diff
@@ -366,9 +422,44 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		msg += "\n\n**Warning:** You subscribed to a private repository. Anyone with access to this channel will be able to read the events getting posted here."
 	}
 
-	return msg
+	post := &model.Post{
+		ChannelId: args.ChannelId,
+		UserId:    p.BotUserID,
+		Message:   msg,
+	}
+
+	if _, appErr := p.API.CreatePost(post); appErr != nil {
+		p.API.LogWarn("error while creating post", "post", post, "error", appErr.Error())
+		return fmt.Sprintf("%s Though there was an error creating the public post: %s", msg, appErr.Error())
+	}
+
+	message := fmt.Sprintf("Successfully subscribed to [%s/%s](%s) with events: %s.", owner, repo, repoLink, subscribeEvents.FormattedString())
```

Comment:
Why is the wording so difference here to org message?

Reply:
@hanzei wanted to keep it unique and make it clear,

can you suggest what could be a better message

Reply:
```suggestion
	message := fmt.Sprintf("Successfully subscribed to [%s/%s](%s) with events: %s.", owner, repo, repoLink, subscribeEvents.FormattedString())
	message := fmt.Sprintf("A subscription to repository [%s/%s](%s) was added to this channel by @%v, with events: %s", owner, repo, repoLink, user.Username, subscribeEvents.FormattedString())
```